Standards/scs-0212-v1-requirements-for-container-registries.md

```diff
@@ -0,0 +1,124 @@
+be pulled from by runtime environments like Kubernetes clusters.
+
+Container registries come in various forms, e.g. publicly accessible ones like Docker Hub or
+self-hosted and cloud-hosted services. The later examples may apply various access control
+mechanisms to restrict access. Both solutions offer a wide range of features that may or may not
+attract potential users and CSPs.
+
+## Terminology
+
+Cloud Service Provider (abbr. CSP)
+  Entity that provides scalable computing resources
+
+## Motivation
+
+This standard is motivated by different use cases identified through the topics in the SCS project.
+One use case would be the offering of private registries for customers by CSPs, which means that a CSP could
+offer private container registries either as a service or as a provided "recipe" to deploy a private registry
+utilizing the CSPs infrastructure.
+Another use case would be the selection of a registry for the SCS reference implementation.
+
+The idea and purpose of this document is to specify what requirements a specific technical container
+registry implementation (i.e. software solution) needs to fulfill for an SCS-compliant registry.
+
+## Design considerations
+
+There are numerous features that should be evaluated for a container registry solution.
+It is important to assess the registries based on the requirements of the OSS health checks and
+desired features defined by the SCS. The following two subsections show these requirements.
+
+### OSS health check
+
+It is important to evaluate the health of a project before even evaluating the project for its feature set.
+A project should therefore fulfill all OSS health checks be defined by the
+[OSS-Health](https://github.com/SovereignCloudStack/standards/blob/main/Drafts/OSS-Health.md) document.
+This document evaluates the health of the open-source projects that were selected from
+the currently available solutions. The container registry software must fulfill all OSS
+health checks defined below:
+
+- Four Opens (code is fully open source, community is open and diverse, development process is open, design process is open)
+- Maturity
+- Security
+- Activity
+- Lock-in risk assessment
+
+### Required and desirable features check
+
+A container registry provides a specific feature set, which can be mapped out against the requirements
+described in this section. The registry should generally be robust (e.g. operate under heavy load) and
+secure in order to be acceptable for the SCS standard. Therefore, a required and optional feature set
+were defined, to which a container registry must abide and be evaluated against.
+
+#### Required features
+
+- Audit Logs
+  - ability to record use in auditable logs so that activity can be traced to a single user
+- Authentication of system identities
+  - support for authenticating system identities like Kubernetes clusters
+  - possibly support for dynamic identity tokens from some IdP
```

Comment:
add IdP to the terminology section

Reply:
Changed in https://github.com/SovereignCloudStack/standards/pull/458/commits/e73ed463c4d03e128dcfa7f1437bfa7131de3f93